src/components/AddToPlaylist.vue

```diff
@@ -79,9 +88,13 @@ export default {
     ...mapGetters("auth", ["currentUser"]),
     ...mapGetters("playlists", ["editablePlaylists"]),
     playlistOptions() {
-      return this.editablePlaylists.filter(
-        (p) => p.playlist_type === this.type
-      );
+      return this.editablePlaylists.reduce((options, p) => {
+        if (p.playlist_type === this.type) {
+          p.disabled = p.item_ids.includes(this.item.id);
```

Comment:
Doesn't this edit the playlist in the store?

Reply:
fixed

Reply:
Not actually fixed, `[... ]` performs a shallow copy.